srv/components/controller/files/scripts/provision.sh

```diff
@@ -1010,6 +1023,55 @@ ftp_enable()
     } && echo "ftp_enable(): ftp is already enabled" >> $logfile
 }
 
+# Check if sftp service is enabled on controller
+is_sftp_enabled()
+{
+    _tmp_file="$tmpdir/is_sftp_enabled"
+    [ -f $_tmp_file ] && rm -rf $_tmp_file
+    # objects name in the xml
+    _xml_obj_bt="security-communications-protocols"
+    _xml_obj_plist=("sftp")
+    # run command to get the details of advanced settings params
+    echo "is_sftp_enabled():Getting protocols details.." >> $logfile
+    _cmd="show protocols"
+    cmd_run "$_cmd"
+    echo "is_sftp_enabled():Checking if sftp service is enabled" >> $logfile
+
+    # parse xml to get required values of properties
+    parse_xml $xml_doc $_xml_obj_bt "${_xml_obj_plist[@]}" > $_tmp_file
+    [ -s $_tmp_file ] || {
+        echo "is_sftp_enabled(): No sftp setting found" >> $logfile
+        rm -rf $_tmp_file
+        return 0
```

Comment:
Returning without printing "Enabled" or "Disabled". Function spec needs to be defined w.r.t. return semantics. Refer to earlier comments.

Reply:
yes, now addressed.